gm4_balloon_animals/data/gm4_balloon_animals/functions/wandering_trader/llama/kill.mcfunction

```diff
@@ -0,0 +1,13 @@
+# Correctly found trader llama to be killed
+# @s = a llama within 5 blocks of the wandering trader being checked
+# at @s
+# from gm4_balloon_animals:wandering_trader/llama/check_leasher
+
+execute on leasher at @s unless entity @e[type=marker,tag=gm4_balloon_animal_item_cleanup,distance=..0.01,limit=1] summon marker run tag @s add gm4_balloon_animal_item_cleanup
+
+scoreboard players add $llamas_replaced gm4_balloon_animals_data 1
+
+tp @s ~ ~-1000 ~
```

Comment:
Similar to my comment elsewhere in this PR, it may be safer to tp to absolute-y -4096.